BLE_Advertising/CMakeLists.txt

```diff
@@ -39,6 +39,8 @@ target_link_libraries(${APP_TARGET}
         mbed-ble-cordio
         mbed-ble-blue_nrg
         mbed-os-ble-utils
+        mbed-mbedtls-cryptocell310
+        mbed-ble-cordio_ll
```

Comment:
what's the diff between `mbed-ble-cordio` and `mbed-ble-cordio_ll` ? I assume only one should be needed ?

Reply:
The build was failed but let me try one more time without mbed-ble-cordio_ll

Reply:
It expects some header files from mbed-ble-cordio_ll
```
./mbed-os/connectivity/drivers/ble/FEATURE_BLE/TARGET_NORDIC/TARGET_NORDIC_CORDIO/TARGET_NRF5x/stack/sources/pal_timer.c:59:10: fatal error: sch_api.h: No such file or directory
   59 | #include "sch_api.h"
      |          ^~~~~~~~~~~

```

Reply:
I wonder if this dep should not be fixed in our Mbed tree rather than here

Reply:
cordio_ll is the link layer, it depends on the board, nrf uses it, others use the HCI driver because their link layer is provided separately

Reply:
This should be fixed in Mbed Os. @rajkan01 please remove these and fix it in ble component.